atem_dissector.lua

```diff
@@ -2226,85 +2232,95 @@ function atem_proto.dissector(tvbuf,pktinfo,root)
-			cmd_tree:add(pf_fields["pf_field_premultiplied"], tvbuf:range(pos+17, 1))
-			cmd_tree:add(pf_fields["pf_field_clip"], tvbuf:range(pos+18, 2))
-			cmd_tree:add(pf_fields["pf_field_gain0"], tvbuf:range(pos+20, 2))
-			cmd_tree:add(pf_fields["pf_field_invertkey0"], tvbuf:range(pos+22, 1))
-			cmd_tree:add(pf_fields["pf_field_borderenabled"], tvbuf:range(pos+23, 1))
-			cmd_tree:add(pf_fields["pf_field_borderbevel"], tvbuf:range(pos+24, 1))
-			cmd_tree:add(pf_fields["pf_field_unknown1"], tvbuf:range(pos+25, 1))
-			cmd_tree:add(pf_fields["pf_field_borderouterwidth"], tvbuf:range(pos+26, 2))
-			cmd_tree:add(pf_fields["pf_field_borderinnerwidth"], tvbuf:range(pos+28, 2))
-			cmd_tree:add(pf_fields["pf_field_borderoutersoftness"], tvbuf:range(pos+30, 1))
-			cmd_tree:add(pf_fields["pf_field_borderinnersoftness"], tvbuf:range(pos+31, 1))
-			cmd_tree:add(pf_fields["pf_field_borderbevelsoftness"], tvbuf:range(pos+32, 1))
-			cmd_tree:add(pf_fields["pf_field_borderbevelposition"], tvbuf:range(pos+33, 1))
-			cmd_tree:add(pf_fields["pf_field_borderhue"], tvbuf:range(pos+34, 2))
-			cmd_tree:add(pf_fields["pf_field_bordersaturation"], tvbuf:range(pos+36, 2))
-			cmd_tree:add(pf_fields["pf_field_borderluma"], tvbuf:range(pos+38, 2))
-			cmd_tree:add(pf_fields["pf_field_lightsourcedirection"], tvbuf:range(pos+40, 2))
-			cmd_tree:add(pf_fields["pf_field_lightsourcealtitude"], tvbuf:range(pos+42, 1))
-			cmd_tree:add(pf_fields["pf_field_unknown1"], tvbuf:range(pos+43, 1))
+			local cmd_cssc_setmask22_tree = cmd_tree:add(pf_fields["pf_cmd_cssc_setmask22"], tvbuf:range(pos+8, 1))
+			cmd_cssc_setmask22_tree:add(pf_fields["pf_flag_cmd_cssc_setmask22_fillsource"], tvbuf:range(pos+8, 1))
+			cmd_cssc_setmask22_tree:add(pf_fields["pf_flag_cmd_cssc_setmask22_keysource"], tvbuf:range(pos+8, 1))
+			cmd_cssc_setmask22_tree:add(pf_fields["pf_flag_cmd_cssc_setmask22_foreground"], tvbuf:range(pos+8, 1))
+			cmd_cssc_setmask22_tree:add(pf_fields["pf_flag_cmd_cssc_setmask22_premultiplied"], tvbuf:range(pos+8, 1))
+			cmd_cssc_setmask22_tree:add(pf_fields["pf_flag_cmd_cssc_setmask22_clip"], tvbuf:range(pos+8, 1))
+			cmd_cssc_setmask22_tree:add(pf_fields["pf_flag_cmd_cssc_setmask22_gain"], tvbuf:range(pos+8, 1))
+			cmd_cssc_setmask22_tree:add(pf_fields["pf_flag_cmd_cssc_setmask22_invert"], tvbuf:range(pos+8, 1))
+			cmd_tree:add(pf_fields["pf_field_ssrc_id"], tvbuf:range(pos+9, 1))
+			cmd_tree:add(pf_fields["pf_field_fillsource"], tvbuf:range(pos+10, 2))
+			cmd_tree:add(pf_fields["pf_field_keysource"], tvbuf:range(pos+12, 2))
+			cmd_tree:add(pf_fields["pf_field_foreground"], tvbuf:range(pos+14, 1))
+			cmd_tree:add(pf_fields["pf_field_premultiplied"], tvbuf:range(pos+15, 1))
+			cmd_tree:add(pf_fields["pf_field_clip"], tvbuf:range(pos+16, 2))
+			cmd_tree:add(pf_fields["pf_field_gain0"], tvbuf:range(pos+18, 2))
+			cmd_tree:add(pf_fields["pf_field_invertkey0"], tvbuf:range(pos+20, 1))
+			cmd_tree:add(pf_fields["pf_field_padding"], tvbuf:range(pos+21, 3))
```

Comment:
```suggestion
			cmd_tree:add(pf_fields["pf_field_unknown1"], tvbuf:range(pos+21, 3))
```
See above, why not using existing unknown field instead of new padding field?